webapp/tests/karma/unit/services/session.js

```diff
@@ -74,16 +74,14 @@ describe('Session service', function() {
     done();
   });
 
-  it('redirects to login if not logged in remotely', function(done) {
+  it('displays prompt if session expires', function(done) {
```

Comment:
So this test doesn't actually test that a prompt is displayed, right?

It's unclear to me if you can do that in a karma test: take a look to see if there are others, but if you can't probably name this test something else that better expresses what its' testing.